Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -150,6 +150,16 @@
             <Style TargetType="uc:DataGridHeader">
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
             </Style>
+
+            <Style
+                x:Key="ColumnContentTextBlock"
+                BasedOn="{StaticResource CaptionTextBlockStyle}"
```

Comment:
I'm not sold to this change in DetailsView: CaptionTextBlockStyle looks _really_ small and seeing two different font sizes on the same row does not look good to me, but I will bow to the designers opinion.. @yaichenbaum? xD

Reply:
Windows 11 introduced Segoe UI variable font which is much more legible at small sizes then old segoe ui. This change is intended to leverage that. Perhaps the old style should be used on windows 10? 

Variable vs standard at size 12:
![image](https://user-images.githubusercontent.com/59544401/128905801-8becc6f0-e305-42f8-a38b-f96265d2cc4a.png)

Reply:
@gave92 I don't have any preference here so I think we should probably use what the design specs suggest.